docs/source/conf.py

```diff
@@ -336,6 +336,7 @@ def run(self):
 
 import torch
 from torch import nn, optim
+from torch.nn.functional import nll_loss
```

Comment:
```suggestion
```

See below, no need to introduce this import.

Reply:
Oops! Thank you!!